web/impact/impact/v1/views/impact_view.py

```diff
@@ -1,48 +1,68 @@
 from abc import (
     ABCMeta,
-    abstractmethod,
+    # abstractmethod,
 )
 from rest_framework.views import APIView
 from rest_framework_tracking.mixins import LoggingMixin
+from impact.permissions import (
+    V1APIPermissions,
```

Comment:
can probably be imported without parenthesis

Reply:
Done.